common/app/common/TrailsToRss.scala

```diff
@@ -123,20 +121,21 @@ object TrailsToRss extends implicits.Collections {
   }
 
   def fromPressedPage(pressedPage: PressedPage)(implicit request: RequestHeader) = {
-    val faciaContentList: List[FaciaContent] =
+    val faciaContentList: List[ContentType] =
       pressedPage.collections
         .filterNot(_.config.excludeFromRss)
         .flatMap(_.curatedPlusBackfillDeduplicated)
         .filter{
           case _: LinkSnap => false
           case _ => true}
-        .filter(_.maybeContentId.isDefined)
-        .distinctBy(faciaContent => faciaContent.maybeContentId.getOrElse(faciaContent.id))
+        .filter(_.properties.maybeContentId.isDefined)
+        .distinctBy(faciaContent => faciaContent.properties.maybeContentId.getOrElse(faciaContent.card.id))
+        .flatMap(_.properties.maybeContent)
```

Comment:
`maybeContent` is an Option, so it would look a bit messy, like `maybeContent.map(_.metadata.id)`. I copied most of those values from facia scala client, which has `maybeContentId`